src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/WpfBasedPropertyPage.cs

```diff
@@ -44,7 +44,9 @@ protected override async Task OnSetObjectsAsync(bool isClosing)
 
         protected override Task<int> OnApplyAsync()
         {
+#pragma warning disable VSTHRD110 // Observe result of async calls
             return _control?.ApplyAsync() ?? Task.FromResult((int)HResult.Fail);
+#pragma warning restore VSTHRD110 // Observe result of async calls
```

Comment:
Is this suppression actually needed? If so, I think it might be a bug in the analyzer.

Reply:
Please remove and do in a followup change if its needed.